pkg/chains/chains.go

```diff
@@ -17,6 +17,7 @@ var (
 		Consensus:   Consensus_tendermint,
 		IsExternal:  false,
 		CctxGateway: CCTXGateway_zevm,
+		Name:        "zeta_mainnet",
 	}
```

Comment:
Why do we need to have hardcoded list here? Do these names match project-wise?

Reply:
>Do these names match project-wise?

What do you mean by this? We hardcode these static chain info because they are not supposed to ever change